ZupIT/ritchie-cli
# Added commons repository acceptance info to metrics

**- What I did**

Added a new bool field to metrics struct
Little refactor at init strings

**- How to verify it**

Use `rit init` , accept, or not the metrics question, then check for metrics on metabse, or log the struct.


**- Description for the changelog**

Added a new bool field to metrics struct
